app-android/src/main/java/io/github/droidkaigi/confsched2023/KaigiApp.kt

```diff
@@ -268,6 +273,30 @@ private class ExternalNavController(
         }
     }
 
+    /**
+     * Navigate to Calendar Registration
+     * @param timeTableItem カレンダー登録に必要なタイムラインアイテムの情報
+     */
+    fun navigateToCalendarRegistration(timeTableItem: TimetableItem) {
```

Comment:
[nits]
How about `navigateToCalendar`?

Reply:
I think this is correct because it jumps to the registration page, not the calendar.

Reply:
Ah, ok. That makes sense.
Thank you for sharing your thought! Let's keep the name as it is 👍